CHANGELOG.rst

```diff
@@ -2,6 +2,16 @@
 CHANGELOG
 =========
 
+2.7.1
```

Comment:
Since this release contains some improvements and new features like no-op trace ids, shouldn't this be a minor version release (2.8.0)?

Reply:
Ahh right, I didn't notice that new features are introduced in the following up PRs as previously we're planned to do a patch release.